docs-new/docs/graphql/core/api-reference/syntax-defs.mdx

```diff
@@ -241,229 +213,204 @@ String
 
-## ArrRelUsingFKeyOn
+## ArrRelUsingFKeyOn {#arrrelusingfkeyon}
 
 | Key    | Required | Schema      | Description                                    |
 |--------|----------|-------------|------------------------------------------------|
-| table  | true     | `TableName` | Name of the table                              |
-| column | true     | `PGColumn`  | Name of the column with foreign key constraint |
+| table  | true     | [TableName](#tablename) | Name of the table                              |
+| column | true     | [PGColumn](#pgcolumn)  | Name of the column with foreign key constraint |
 
-## ArrRelUsingManualMapping
+## ArrRelUsingManualMapping {#arrrelusingmanualmapping}
 
 | Key            | Required | Schema                           | Description                                               |
 |----------------|----------|----------------------------------|-----------------------------------------------------------|
-| remote_table   | true     | `TableName`                      | The table to which the relationship has to be established |
-| column_mapping | true     | Object (`PGColumn` : `PGColumn`) | Mapping of columns from current table to remote table     |
+| remote_table   | true     | [TableName](#tablename)                      | The table to which the relationship has to be established |
+| column_mapping | true     | Object ([PGColumn](#pgcolumn) : [PGColumn](#pgcolumn)) | Mapping of columns from current table to remote table     |
 
-## BoolExp
+## BoolExp {#boolexp}
 
 <div class="parsed-literal haskell-pre">
 
 [AndExp](#andexp) \| [OrExp](#orexp) \| [NotExp](#notexp) \|
 [ExistsExp](#existsexp) \| [TrueExp](#trueexp) \|
-[ColumnExp](#columnexp)
+ColumnExp](#columnexp)
 
 </div>
 
-## AndExp
+## AndExp {#andexp}
 
 <div class="parsed-literal haskell-pre">
 
 {  
-"$and" : \[[BoolExp](#boolexp)\],
-
+    "$and" : \[[BoolExp](#boolexp)\],
 }
 
 </div>
 
-## OrExp
+## OrExp {#orexp}
 
 <div class="parsed-literal haskell-pre">
 
 {  
-"$or" : \[[BoolExp](#boolexp)\],
-
+    "$or" : \[[BoolExp](#boolexp)\],
 }
 
 </div>
 
-## NotExp
+## NotExp {#notexp}
 
 <div class="parsed-literal haskell-pre">
 
 {  
-"$not" : [BoolExp](#boolexp)
-
+    "$not" : [BoolExp](#boolexp)
 }
 
 </div>
 
-## ExistsExp
+## ExistsExp {#existsexp}
 
 <div class="parsed-literal haskell-pre">
 
 {  
-"$exists" : {  
-"\_table": [TableName](#tablename), "\_where": [BoolExp](#boolexp)
-
-}
-
+    "$exists" : {  
+        "\_table": [TableName](#tablename), "\_where": [BoolExp](#boolexp)
+    }
 }
 
 </div>
 
-## TrueExp
+## TrueExp {#trueexp}
 
-<div class="parsed-literal haskell-pre">
+`{}`
 
-{}
-
-</div>
-
-## ColumnExp
+## ColumnExp {#columnexp}
 
 <div class="parsed-literal haskell-pre">
 
 {  
 [PGColumn](#pgcolumn) \| scalar [ComputedFieldName](#computedfieldname)
 : { [Operator](#operator) : Value }
 
-}
-
 </div>
 
-## Operator
+## Operator {#operator}
```

Comment:
is mismatch from original rst file

`{#metadataoperator}`

please update respective usage in this file

Reply:
Fixed